packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -104,8 +109,15 @@ export function getPropGetters<
       role: 'combobox',
       'aria-expanded': store.getState().isOpen,
       'aria-haspopup': 'listbox',
-      'aria-owns': store.getState().isOpen ? `${props.id}-list` : undefined,
-      'aria-labelledby': `${props.id}-label`,
+      'aria-owns': store.getState().isOpen
+        ? store
+            .getState()
+            .collections.map(({ source }) =>
+              getAutocompleteElementId(props.id, 'list', source)
```

Comment:
bit unfortunate that this gets calculated again instead of saved in the collection, but it makes sense